charts/kubearchive/templates/certificates.yaml

```diff
@@ -3,32 +3,32 @@
 apiVersion: cert-manager.io/v1
 kind: Issuer
 metadata:
-  name: kubearchive-ca-issuer
+  name: "{{ .Release.Name }}-ca"
 spec:
   selfSigned: {}
 
 ---
 apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
-  name: kubearchive-ca-certificate
+  name: "{{ .Release.Name }}-ca"
 spec:
   isCA: true
   commonName: kubearchive-ca-certificate
-  secretName: kubearchive-ca-secret
+  secretName: "{{ .Release.Name }}-ca"
   privateKey:
     algorithm: ECDSA
     size: 256
   issuerRef:
-    name: kubearchive-ca-issuer
+    name: "{{ .Release.Name }}-ca"
     kind: Issuer
     group: cert-manager.io
 
 ---
 apiVersion: cert-manager.io/v1
 kind: Issuer
 metadata:
-  name: kubearchive-cert-issuer
+  name: "{{ .Release.Name }}"
 spec:
   ca:
-    secretName: kubearchive-ca-secret
+    secretName: "{{ .Release.Name }}-ca"
```

Comment:
`{{ .Release.Name }}-ca` appears 5 times here. Based on that and on how we refactored the other helm templates I think we should include a variable in the values with that template and use `tpl` to reference it.

Something like:

values.yaml
```yaml
kubearchive:
  ca: "{{ .Release.Name }}-ca"
...
```

certificates.yaml
```yaml
...
spec:
  ca:
    secretName: "{{ tpl .Values.kubearchive.ca }}"
...
```

Reply:
Instead of creating a new variable in the values, I created the variable at the top of the file with an assignment. This is not a variable for the user to modify.